frontend/src/modules/Shares/services/rejectShareObject.js

```diff
@@ -1,6 +1,6 @@
 import { gql } from 'apollo-boost';
 
-const rejectShareObject = ({ shareUri, rejectPurpose }) => ({
+export const rejectShareObject = ({ shareUri, rejectPurpose }) => ({
```

Comment:
After the merge, `rejectShareObject` have ceased to comply with updated lint rules.

Reply:
And the same for all new added share services

Reply:
I added a TODO already when merging, and fixed linting issues for newly added files. Not quite sure how this file doesn't comply with  linting rules.

Reply:
It can be easily reproduced if you run the linting command in the file `deploy/stacks/pipeline.py` in this PR. For me, it is throwing 210 problems